docs/TLS.md

```diff
@@ -97,6 +97,10 @@ you will find that they are valid and short-term.
 
 ## Generate certificates manually
 
+!!! warning
+
+    Manual certificates generation doesn't work in the Operator version 1.16.0.
```

Comment:
i'd say something like:

"Using manually generated certificates doesn't work due to a bug in the Operator version 1.16.0. Check https://perconadev.atlassian.net/browse/K8SPSMDB-1101 for updates."

Reply:
updated